discordSuperUtils/imaging.py

```diff
@@ -360,6 +362,123 @@ async def create_leveling_profile(
         result_bytes.seek(0)
         return discord.File(result_bytes, filename="rankcard.png")
 
+    async def create_spotify_card(
+        self,
+        spotify_result: discord.Spotify,
+        font_path: str = None
+    ) -> discord.File:
+        """
+        |coro|
+
+        Creates a welcome image for the member and returns it as a discord.File.
+
+        :param discord.Spotify spotify_result: The Spotify activity.
+        :param str font_path: The font path, uses the default font if not passed.
+        :return: The discord file.
+        :rtype: discord.File
+        """
+
+        result_bytes = BytesIO()
+
+        track_background_image = Image.open(
+            ImageManager.load_asset(
+                "spotify_template.png"
+            )
+        )
+        
+        album_image = Image.open(
```

Comment:
This is good, but we already have a function for that, please look at ImageManager.convert_image, does the exact same thing.